R/zzz.R

```diff
@@ -1,5 +1,27 @@
 .lockbox_env <- new.env()
 
+.get_safely_from_envir <- function(target, default = NULL) {
+  res <- tryCatch(
+    get(target, envir = .lockbox_env)
+  , error = function(e) {default})
```

Comment:
You're right.  I don't know why I got in the pattern of get() for a target environment rather than using `$`... which kindly returns a NULL anyway rather than throwing an error.